modules/stochastic_tools/src/reporters/EvaluateSurrogate.C

```diff
@@ -0,0 +1,116 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// Stocastic Tools Includes
+#include "EvaluateSurrogate.h"
+
+#include "Sampler.h"
+
+registerMooseObject("StochasticToolsApp", EvaluateSurrogate);
+
+InputParameters
+EvaluateSurrogate::validParams()
+{
+  InputParameters params = StochasticReporter::validParams();
+  params += SurrogateModelInterface::validParams();
+  params.addClassDescription("Tool for sampling surrogate models.");
+  params.addRequiredParam<std::vector<UserObjectName>>("model", "Name of surrogate models.");
+  params += SamplerInterface::validParams();
```

Comment:
```suggestion
  params += SurrogateModelInterface::validParams();
  params += SamplerInterface::validParams();
  params.addClassDescription("Tool for sampling surrogate models.");
  params.addRequiredParam<std::vector<UserObjectName>>("model", "Name of surrogate models.");
```

Yes - the ordering does bother me 😆